pkg/registry/apis/identity/team/store_members.go

```diff
@@ -0,0 +1,111 @@
+package team
+
+import (
+	"context"
+	"net/http"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	"github.com/grafana/authlib/claims"
+	"github.com/grafana/grafana/pkg/api/dtos"
+	identityv0 "github.com/grafana/grafana/pkg/apis/identity/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/common"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/legacy"
+	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+var (
+	_ rest.Storage         = (*LegacyTeamMemberStore)(nil)
```

Comment:
```suggestion
```
I don't think this should/needs to implement Storage -- just the connector stuff

Reply:
We need to implement store, otherwise we cant register it https://github.com/grafana/grafana/blob/main/pkg/registry/apis/identity/register.go#L81